maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/WebServiceTestObjectUnitTest.java

```diff
@@ -128,7 +127,7 @@ private Supplier<HttpClient> getHttpClientSupplier() {
    *
    * @return a web service driver.
    */
-  private WebServiceDriver getWebServiceDriver() {
+  private WebServiceDriver getWebServiceDriverWithURI() {
```

Comment:
Why did we rename this?
I am fine with it changing this is wordier and the casing is funky.
If you are keeping this name change, may I suggest  getWebServiceDriverWith**Uri** instead?

Reply:
changed the name back, was resolved with reference changed back to generictest